utils/messages.js

```diff
@@ -504,8 +521,7 @@ function buildMessageClass(details) {
             new (field.messageType)(values ? values[field.name] : undefined);
         } else {
           // simple value
-          that[field.name] = values ? values[field.name] :
-            (field.value || fieldsUtil.getDefaultValue(field.type));
+          that[field.name] = values ? values[field.name] : null;
```

Comment:
Won't this force users to specify values for all the fields in their messages?
```
const header = new std_msgs.msg.Header()
pub.publish(header); // invalid
```
whereas before that would have worked